unshared_currency_multicompany/currency.py

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#  licence AGPL version 3 or later
+#  see licence in __openerp__.py or http://www.gnu.org/licenses/agpl-3.0.txt
+#  Copyright (C) 2014 Akretion (http://www.akretion.com).
+#  @author David BEAL <david.beal@akretion.com>
+#
+##############################################################################
+
+from openerp.osv import orm, fields
+
+
+class ResCurrency(orm.Model):
+    _inherit = 'res.currency'
+
+    def name_get(self, cr, uid, ids, context=None):
+        """ If currencies are not shared and they have the same name,
+            then display have to distinguish them"""
+        result = []
+        for currency in self.browse(cr, uid, ids, context):
+            cpny_info = ''
+            if currency.company_id:
+                cpny_info = ' (%s)' % currency.company_id.name
```

Comment:
sorry if that's a dumb question, but why use that 1st interpolation here (' (%s)' %) ?
can you not just do:
result.append((currency.id, '%s %s' % (currency.name, currency.company_id.name)))
?